### PR TITLE
calldata rendering bugfix, large memory offset bugfix

### PR DESCRIPTION
**Pull request by karmacoma-eth**

### CODE REVIEW

File: src/halmos/__main__.py [hunk 1]
```diff
@@ -237,6 +237,15 @@ def rendered_trace(context: CallContext) -> str:
         return output.getvalue()
 
 
+def rendered_calldata(calldata: List[Byte]) -> str:
+    if any(is_bv(x) for x in calldata):
+        # make sure every byte is wrapped
+        calldata_bv = [x if is_bv(x) else con(x, 8) for x in calldata]
+        return hexify(simplify(Concat(calldata_bv)))
```
Comment (daejunpark) [inline src/halmos/__main__.py:244]: `Concat` will fail if the given list is a singleton. `concat` should be used.

Comment (karmacoma-eth) [reply src/halmos/__main__.py:244]: Concat, why are you so finicky? 🙈

Comment (karmacoma-eth) [reply src/halmos/__main__.py]: in c430f4e

File: src/halmos/sevm.py [hunk 2]
```diff
@@ -39,6 +39,8 @@
 EMPTY_BYTES = b""
 MAX_CALL_DEPTH = 1024
 
+# TODO: make this configurable
+MAX_MEMORY_SIZE = 2**20
```
Comment (daejunpark) [inline src/halmos/sevm.py:43]: can this be increased to 2^32 too?

**Review by daejunpark — APPROVED** [review]

**Pull request merged**